packages/cf-forms/src/molecules/form-fields.less

```diff
@@ -11,6 +11,16 @@
     &__checkbox,
     &__radio {
         .a-label {
+            // We need to turn off autoprefixing for the inline-grid because
+            // IE does not handle an inline-grid like other browsers,
+            // leading to an extremely narrow column of text for the label
+            // and the checkbox or radio widget covering the first part of it.
+            // The Autoprefixer ontrol comment below ensures that the following
```

Comment:
Noting for the next minor update: `ontrol` --> `control`

Reply:
Nooooooooooo!